Auto update failing for kakoune

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [kakoune](https://github.com/termux/termux-packages/blob/54d6aa90951d9709f1b867cfa08867b71ce31e7b/packages/kakoune) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating kakoune [Current version: 2023.08.05]
INFO: package being updated to 2024.05.09.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  574k  100  574k    0     0  2613k      0 --:--:-- --:--:-- --:--:-- 2613k

You are about to commit these changes:

--------------------
diff --git a/packages/kakoune/build.sh b/packages/kakoune/build.sh
index 501d8cb28..7beeb640f 100644
--- a/packages/kakoune/build.sh
+++ b/packages/kakoune/build.sh
@@ -2,10 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/mawww/kakoune
 TERMUX_PKG_DESCRIPTION="Code editor heavily inspired by Vim"
 TERMUX_PKG_LICENSE="Unlicense"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="2023.08.05"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="2024.05.09"
 TERMUX_PKG_SRCURL=https://github.com/mawww/kakoune/releases/download/v$TERMUX_PKG_VERSION/kakoune-$TERMUX_PKG_VERSION.tar.bz2
-TERMUX_PKG_SHA256=3e45151e0addd3500de2d6a29b5aacf2267c42bb256d44a782e73defb29cda5c
+TERMUX_PKG_SHA256=2190bddfd3af590c0593c38537088976547506f47bd6eb6c0e22350dbd16a229
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="libc++"
 TERMUX_PKG_BUILD_IN_SRC=true
--------------------

bump(main/kakoune): 2024.05.09


Not committing to Git!

INFO: Trying to build package.
INFO: Package kakoune exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building kakoune for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13133  100 13133    0     0  36581      0 --:--:-- --:--:-- --:--:-- 36582
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2247      0 --:--:-- --:--:-- --:--:--  2251
gpg: Signature made Thu 09 May 2024 10:55:45 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13319  100 13319    0     0  36538      0 --:--:-- --:--:-- --:--:-- 36590
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2251      0 --:--:-- --:--:-- --:--:--  2251
gpg: Signature made Wed 08 May 2024 01:36:55 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13119  100 13119    0     0  80656      0 --:--:-- --:--:-- --:--:-- 80981
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2431      0 --:--:-- --:--:-- --:--:--  2435
gpg: Signature made Thu 09 May 2024 10:56:22 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libc++@26b if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/mawww/kakoune/releases/download/v2024.05.09/kakoune-2024.05.09.tar.bz2
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  574k  100  574k    0     0  2712k      0 --:--:-- --:--:-- --:--:-- 2712k
Applying patch: src-cs-path-compat.patch
Applying patch: src-main.cc.patch
make: *** No targets specified and no makefile found.  Stop.
make: Entering directory '/home/builder/.termux-build/kakoune/src/src'
make: Leaving directory '/home/builder/.termux-build/kakoune/src/src'
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-05-09 12:22:10 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/9016980126">9016980126</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/kakoune/Makefile.patch
+++ b/packages/kakoune/Makefile.patch
@@ -1,0 +1,20 @@
+--- a/Makefile
++++ b/Makefile
+@@ -1,7 +1,7 @@
+ .POSIX:
+ .SUFFIXES:
+ 
+-CXX = c++
++CXX ?= c++
+ 
+ debug = no
+ static = no
+@@ -33,7 +33,7 @@
+ version = $(shell cat .version 2>/dev/null || git describe --tags HEAD 2>/dev/null | sed s/^v// || echo unknown)
+ version != cat .version 2>/dev/null || ( git describe --tags HEAD 2>/dev/null | sed s/^v// ) || echo unknown
+ 
+-PREFIX = /usr/local
++PREFIX ?= /usr/local
+ DESTDIR = # root dir
+ 
+ bindir = $(DESTDIR)$(PREFIX)/bin

--- a/packages/kakoune/build.sh
+++ b/packages/kakoune/build.sh
@@ -1,15 +1,13 @@
-TERMUX_PKG_HOMEPAGE=https://github.com/mawww/kakoune
+TERMUX_PKG_HOMEPAGE=https://kakoune.org/
 TERMUX_PKG_DESCRIPTION="Code editor heavily inspired by Vim"
 TERMUX_PKG_LICENSE="Unlicense"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="2023.08.05"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="2024.05.09"
 TERMUX_PKG_SRCURL=https://github.com/mawww/kakoune/releases/download/v$TERMUX_PKG_VERSION/kakoune-$TERMUX_PKG_VERSION.tar.bz2
-TERMUX_PKG_SHA256=3e45151e0addd3500de2d6a29b5aacf2267c42bb256d44a782e73defb29cda5c
+TERMUX_PKG_SHA256=2190bddfd3af590c0593c38537088976547506f47bd6eb6c0e22350dbd16a229
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="libc++"
 TERMUX_PKG_BUILD_IN_SRC=true
-TERMUX_PKG_EXTRA_MAKE_ARGS=" -C src debug=no "
 
 termux_step_create_debscripts() {
 	cat <<- EOF > ./postinst

